readme.md

```diff
@@ -11,9 +11,9 @@ We value your privacy highly. The Interceptor is designed to minimize privacy le
 # Installation
 Download [The Interceptor](https://github.com/DarkFlorist/TheInterceptor/releases/latest) and depending on your browser:
 
-- Chrome: Browse to `chrome://extensions/` and click `Load unpacked` and select the zip.
+- Chrome: Unzip The Interceptor zip to a folder, browse to `chrome://extensions/` and click `Load unpacked` and select the folder.
 - Firefox: Browse to `about:debugging` and click `Load Temporary Add-on` and select the zip.
```

Comment:
Wonder if we should mention you need to do this every time you restart Firefox?

Reply:
good point. I didn't realize you have to do this